build.gradle

```diff
@@ -225,3 +234,20 @@ bootJar {
 wrapper {
     distributionType = Wrapper.DistributionType.ALL
 }
+
+flyway {
+  url = System.getenv('FLYWAY_URL')
+  user = System.getenv('FLYWAY_USER')
```

Comment:
should this use the POSTGRES_USER env var?

Reply:
Done